doc/nrf/known_issues.rst

```diff
@@ -1101,6 +1101,14 @@ NCSDK-17088: :ref:`nrf_desktop_ble_qos` may crash on application start
 
   **Workaround:** Manually cherry-pick and apply the commit with the fix from the ``main`` branch (commit hash: ``f236a8eff32adbe201d486cd11d4fa8853b90bd7``).
 
+.. rst-class:: v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
+
+NCSDK-17706: :ref:`nrf_desktop_failsafe` does not continue interrupted settings erase operation
+  The failsafe module does not continue interrupted settings erase operation on subsequent boot.
+  Becase of that, the application may be booted with partially erased settings.
```

Comment:
```suggestion
NCSDK-17706: :ref:`nrf_desktop_failsafe` does not continue an interrupted settings erase operation
  The failsafe module does not continue an interrupted settings erase operation on a subsequent boot.
  Because of that, the application may be booted with only partially erased settings.
```